integration/kubernetes/init.sh

```diff
@@ -133,6 +133,7 @@ sed -i "s|CGROUP_DRIVER|${cgroup_driver}|" "${kubeadm_config_file}"
 trap 'sudo -E sh -c "rm -r "${kubeadm_config_file}""' EXIT
 
 if [ "${CI}" == true ] && [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
+	cat /proc/swaps | grep -q zram && sudo touch /etc/systemd/zram-generator.conf # turn off zram swap if exist
```

Comment:
Nits: This could be simplified to remove the `cat`, but also, although that is the official way to disable zram, wouldn't it be clearer to write a comment to that file. Something like:

```suggestion
	grep -q zram /proc/swaps && echo "# zram swap disabled"  | sudo tee -a /etc/systemd/zram-generator.conf
```

Reply:
I'll do that, actually i won't even append as if for some reason this file is not empty i guess it won't work